products/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from products.views import ProductListView
+
+urlpatterns = [
+    path('/list', ProductListView.as_view()),
```

Comment:
wesulloc/urls.py에 path 설정이 products로 되어 있기 때문에, `producuts/list` 보다는 `products` 만으로 충분히 목록을 호출하는지 알 수 있을 것 같습니다.
```suggestion
    path('', ProductListView.as_view()),
```

Reply:
`products/list`라고 지정해 둔 이유는
애초에 저희가 클론코딩하는 오설록 웹사이트에서 제품 리스트의 엔드포인트를 따로 지정해두었기 때문이기도 합니다.

또한, 이후 진행한 제품 상세페이지의 엔드포인트를 지정하는 과정에서 `products`만을 사용하는 것보다 제품 리스트와 상세페이지에 각각의 엔드포인트를 지정해주는 것이 더 효과적이라 생각했습니다.

Reply:
좋습니다!
오설록을 따라갈 필요는 없을 것 같아요!

대부분 엔드포인트를 만들 때, `products` 형태로 많이 사용합니다.
대표적인 예로 cafe24 api 문서를 참고해 보시면 좋을 것  같습니다.
https://developers.cafe24.com/docs/api/admin/#products
![스크린샷 2022-05-30 오후 1 53 54](https://user-images.githubusercontent.com/53214510/170919527-1056c132-5778-4f8d-9d40-bf611f7f0338.png)

Reply:
멘토님 피드백 반영하여 링크 주신 api문서 확인 후 `path('', ProductListView.as_view()),`로 수정하였습니다!